### DISCUSSION TITLE
Lonsonho ZB-CL01 (ZB-RGBCW) color bulb can not change brightenes and temperature

### DISCUSSION THREAD

**Discussion by ArturSku** [Koenkk/zigbee2mqtt#16430] [Devices]
I'm new at Home Assistant so sorry for asking but help is really needed (before I get crazy).

I have following issue - I'm not able to change temperature and/or brightness of Lonsonho ZB-CL01 color bulb.
https://www.zigbee2mqtt.io/devices/ZB-RGBCW.html

I. I can switch the bulb on and off only if (service: light.on):
1. color is set at dashboard (all colors and temperatures as well as brightness I can use), condition: no parameters were selected at light.on service
2. color is set based on RGB.

II. After I set any parameters (even: brightness, color_temp, color_xy, as served by bulb)
1. the light do not turn on if it was off, the light turns off by switch and do not turn on again.
2. If bulb was on and I changed parameters it accept new parameters and is on as long as I switch it off. After that it do not switch on.

III. Others
1. Changing settings by addaptive light do not work,
2. Changing patameters by profil also do not work.
3.  Changing colors by calling scenes do not work

Could you, please give me some advise how to solve that concern.

**Comment by yurgh** [comment]
Reviving this post. I've spent hours trying to get the eWeLight ZB-CL01 (firmware version 2.0.1) behave like a normal light. It's just not possible. If transition is not set, it will allow turning on and off without any additional settings (like brightness, color etc) - then need another turn on with the additional data. If transition is set to 1 second it will allow turning on while setting brightness, but it will not turn off again. Not at all, by any method.

A possible workaround could be to turn on the controller, then wait 500ms, then set color/brightness.

Me; I'm handing this in for recycling and getting some more robust controllers...
